### PR TITLE
fixed: Compile warnings

### PR DESCRIPTION
**Pull request by arnova**
## Description

Compile warning fixes
## Types of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
## Checklist:

- [ ] My code follows the [Code guidelines](https://codedocs.xyz/xbmc/xbmc/code_guidelines.html) of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [ ] I have read the [CONTRIBUTING](https://github.com/xbmc/xbmc/blob/master/CONTRIBUTING.md) document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

**Comment by arnova** [comment]
jenkins build this please

File: xbmc/video/VideoInfoScanner.cpp [hunk 1]
```diff
@@ -458,7 +458,7 @@ namespace VIDEO
       if (ret == INFO_CANCELLED || ret == INFO_ERROR)
       {
         CLog::Log(LOGWARNING,
-                  "VideoInfoScanner: Error %s occurred while retrieving"
+                  "VideoInfoScanner: Error %i occurred while retrieving"
```
Comment (garbear) [inline xbmc/video/VideoInfoScanner.cpp]: %u

**Comment by garbear** [comment]
let's wait for #10740 and #10765 (these include both fixes) because the fixes are split and have more descriptive commit messages

File: xbmc/video/VideoInfoScanner.cpp (hunk 1 above)
Comment (arnova) [inline xbmc/video/VideoInfoScanner.cpp]: Ah yes, it's an enum ofc.

**Comment by arnova** [comment]
@garbear : Fine by me, didn't notice there was already a PR for it.

**Comment by MartijnKaijser** [comment]
@arnova ping

**Comment by arnova** [comment]
This is obsolete

**Pull request closed without merging**